### PR TITLE
Use env variable for config location

### PR DESCRIPTION
**Pull request by geek**

### CODE REVIEW

File: customers/Dockerfile [hunk 1]
```diff
@@ -21,7 +21,6 @@ RUN export CP_SHA1=198d96c8d7bfafb1ab6df96653c29701510b833c \
     && rm /tmp/containerpilot.tar.gz
 
 # add our application
-COPY containerpilot.json /opt/customers/
```
Comment (tgross) [inline customers/Dockerfile:24]: This eliminates the file itself, not the environment variable. What are we trying to do here?

Comment (geek) [reply customers/Dockerfile:24]: See https://github.com/geek/workshop/blob/61843f5f84d36e83631932e29740d1b6ea5609d5/customers/Dockerfile#L29

Comment (tgross) [reply customers/Dockerfile:24]: Oh, I see we did it on line 30 below.

**Comment by tgross** [comment]
This should have been merged ages ago.

**Pull request merged**